test/bridge/testhorton.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+import unittest
+import os, sys
+import numpy
+
+from cclib.bridge import cclib2horton
+from ..test_data import getdatafile
+
+
+class HortonTest(unittest.TestCase):
+    """Tests for the horton bridge in cclib."""
+
+    # Both packages are able to read in fchk
```

Comment:
Could you be more specific here? Which "both" packages?

Also, please end sentences in comments with periods.